apps/vite/src/components/calendar/CalendarHeader.tsx

```
@@ -13,73 +22,43 @@ const CalendarHeader: React.FC<{ month: string; year: string }> = ({ month, year
                 <button
-                  className={classNames(
-                    'inline-flex items-center transition-colors duration-300 ease-in focus:outline-none hover:text-blue-400 focus:text-blue-400 rounded-md px-2 py-2',
-                    view === 'event' ? 'bg-white' : ''
+                  className={cn(
+                    "inline-flex items-center transition-colors duration-300 ease-in focus:outline-none hover:text-blue-400 focus:text-blue-400 rounded-md px-2 py-2",
+                    view === "event" ? "bg-white" : ""
                   )}
                   id="list"
-                  onClick={() => setView('event')}
+                  onClick={() => setView("event")}
                 >
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)